codec-http2/src/main/java/io/netty/handler/codec/http2/HpackStaticTable.java

```diff
@@ -145,7 +146,7 @@ static int getIndex(CharSequence name) {
      * Returns the index value for the given header field in the static table. Returns -1 if the
      * header field is not in the static table.
      */
-    static int getIndex(CharSequence name, CharSequence value) {
+    static int getIndexSensitive(CharSequence name, CharSequence value) {
```

Comment:
I don't see any usage of this. And I wouldn't expect any usage since sensitive header values won't use the HPACK table.

Reply:
Yup, no use, removed.